packages/website/pages/docs/reference/prover-market-page.mdx

```diff
@@ -28,3 +28,4 @@ You can edit the page by clicking [here](https://github.com/taikoxyz/taiko-mono/
 | Cryptic Node   | http://65.21.90.249:9876                | 10 wei      |
 | dannyletra     | http://158.220.89.198:9876              | 10 wei      |
 | AlexIT         | http://62.183.54.219:9876               | 10 wei      |
+  ZK Nodes         http://45.144.28.60:9876                  10 wei
```

Comment:
Please add `|` to separate columns on the last row.

Reply:
just changed it. does that work. hopefully i saved
 it lol